bigtable/integration_test.go

```diff
@@ -2190,3 +2193,22 @@ func clearTimestamps(r Row) {
 		}
 	}
 }
+
+func deleteTable(ctx context.Context, t *testing.T, ac *AdminClient, name string) {
+	retryOptions = []gax.CallOption{
+		gax.WithRetry(func() gax.Retryer {
+			return gax.OnCodes(retryCodes, gax.Backoff{
+				Initial:    100 * time.Millisecond,
+				Max:        2 * time.Second,
+				Multiplier: 1.2,
+			})
+		}),
+	}
+	err := gax.Invoke(ctx, func(ctx context.Context, _ gax.CallSettings) error {
```

Comment:
I don't see a deadline for this, do we need to add one to the context or is it taken care of somewhere?

Reply:
I believe Max is the max value of the "retry envelope", i.e. the time between retries. You still have to set a context deadline to avoid retrying indefinitely. See https://github.com/googleapis/gax-go/blob/v2.0.5/v2/call_option.go#L97

Reply:
Yeah, in fact I've read it, but the word "envelope" translates to russian as "a flat paper container with a sealable flap, used to enclose a letter or document", so I just didn't get it correctly :cry:
I thought it equals to retry wrapper on the whole or something like that. Anyway, I've pushed

Reply:
No worries, it is confusing in English as well!